### PR TITLE
updated technical concepts

### PR DESCRIPTION
**Pull request by kennedyuche**
- updated technical concepts

### CODE REVIEW

File: content/en/docs/concepts/technical concepts.md [hunk 1]
```diff
@@ -18,8 +34,47 @@ Beamstack acts as a low-code abstraction layer that allow users to deploy ML pip
...
+  <ul>
+    <li><b>Custom Dashboards</b>: Grafana integrates with Prometheus to offer customizable dashboards where users can visualize collected metrics through various types of charts, graphs, and tables. Users can create and configure dashboards to display key performance indicators (KPIs) and other critical metrics relevant to their ML pipelines.</li>
+  </ul>
+  <ul>
+    <li><b>Interactive Data Exploration</b>: Grafana provides interactive data exploration capabilities, allowing users to drill down into specific metrics, filter data, and generate ad-hoc queries. This interactive functionality aids in detailed analysis and troubleshooting.</li>
+  </ul>
+</details>
+
+<details>
+  <summary><b>Container and Resource Monitoring</b></summary>
```
Comment (charlesa101) [inline content/en/docs/concepts/technical concepts.md:62]: will be good to add the screen shot

File: content/en/docs/concepts/technical concepts.md [hunk 2]
```diff
@@ -28,7 +83,69 @@ Beamstack provides a monitoring stack consisting of Grafana and Prometheus that
...
+  <ul>
+    <li><b>Unified Interface</b>: Beamstack provides a consistent user interface and experience, whether users are operating from different OS environments or using different development setups. This consistency reduces the learning curve and accelerates the development and deployment process.</li>
+  </ul>
+  <ul>
+    <li><b>Documentation and Support</b>: Comprehensive documentation and support resources are available to assist users across different environments. This includes platform-specific guidelines, troubleshooting tips, and best practices to ensure smooth operation regardless of the user's setup.</li>
+  </ul>
+</details>
+
+<details>
+  <summary><b>Scalability and Portability</b></summary>
```
Comment (charlesa101) [inline content/en/docs/concepts/technical concepts.md]: this should be another section on its own

File: content/en/docs/concepts/technical concepts.md (hunk 1 above)
Comment (olufunbi) [reply content/en/docs/concepts/technical concepts.md:62]: monitoring
@kennedyuche You can use this, feel free to re-size.

**Review by olufunbi — CHANGES_REQUESTED** [review]
Looks good, all that is left is the screenshot change.

**Review by olufunbi — APPROVED** [review]

**Pull request merged**